event_webhook_types_test.go

```diff
@@ -230,23 +229,23 @@ func TestMergeEventUnmarshal(t *testing.T) {
 	}
 
 	if event.Assignees[0].AvatarURL != excpectedAvatar {
-		t.Errorf("Assignees[0].Email is %v, want %v", event.Assignees[0].AvatarURL, excpectedAvatar)
+		t.Errorf("Assignees[0].AvatarURL is %v, want %v", event.Assignees[0].AvatarURL, excpectedAvatar)
 	}
 
-	if event.Assignees[0].Email != expectedEmail {
-		t.Errorf("Assignees[0].Email is %v, want %v", event.Assignees[0].Email, expectedEmail)
+	if len(event.Reviewers) < expectedID {
+		t.Errorf("Reviewers length is %d, want %d", len(event.Reviewers), expectedID)
 	}
 
-	if len(event.Changes.Reviewers.Previous) != 0 {
-		t.Errorf("Changes.Reviewer.Previeous length is %d, want %d", len(event.Changes.Reviewers.Previous), 0)
+	if event.Reviewers[0].Name != expectedName {
+		t.Errorf("Reviewers[0].Name is %v, want %v", event.Reviewers[0].Name, expectedName)
```

Comment:
Instead of changing the existing tests, please add new ones if you want to add more tests. So please make sure all the existing tests aren't changed or removed.

Reply:
Same reason as above, please let me know if you would like to the current documentation's payload or alter it to meet the tests.